scripts/performmotioninputs/performmotioninputs.gml

```diff
@@ -0,0 +1,120 @@
+// Script assets have changed for v2.3.0 see
+// https://help.yoyogames.com/hc/en-us/articles/360005277377 for more information
+
+// Allows player to perform motion inputs during special moves
+function PerformMotionInputs()
+{
+	if (inputSet)
+	{
+		var inputPerformed = false;
+		
+		var inputDirection = 5; // Neutral; Default
+		// Set input directions
+		if (movedir == -image_xscale && verticalMoveDir == -1)
+		{
+			inputDirection = 1; // Down-back
+		}
+		else if (movedir == 0 && verticalMoveDir == -1)
+		{
+			inputDirection = 2; // Down
+		}
+		else if (movedir == image_xscale && verticalMoveDir == -1)
+		{
+			inputDirection = 3; // Down-forward
+		}
+		else if (movedir == -image_xscale && verticalMoveDir == 0)
+		{
+			inputDirection = 4; // Back
+		}
+		else if (movedir == 0 && verticalMoveDir == 0)
+		{
+			inputDirection = 5; // Neutral
```

Comment:
Technically unnecessary

Reply:
Neutral input is used for inputs like 22 or 66. This way, the player has to input the direction twice and it prevents the input from being performed immediately. The code changes these types of inputs behind the door to 252 and 656 respectively for this purpose.

Reply:
understood